package.json

```diff
@@ -10,7 +10,7 @@
 		"url": "github.com/kevva"
 	},
 	"engines": {
-		"node": ">=4"
+		"node": ">=7.6.0"
```

Comment:
So, the idea is not supporting legacy node anymore?

Reply:
For the bugfix (minor/patch version) the node version shouldn't change. But for a major release, I would skip to [node 10 since it's the latest LTS](https://github.com/nodejs/Release#release-schedule).

Reply:
Yeah if somebody can get things running in Node 4, that’s cool. I couldn’t, likely due to the wildcard xo and ava deps.